packages/audioplayers/tizen/src/audio_player.cc

```diff
@@ -0,0 +1,484 @@
+#include "audio_player.h"
+
+#include "audio_player_error.h"
+#include "log.h"
+
+static std::string StateToString(player_state_e state) {
```

Comment:
Unused function? Even if it is used, the same thing can be done using just [stringification](https://gcc.gnu.org/onlinedocs/gcc-4.8.5/cpp/Stringification.html).